src/UKHO.ERPFacade.Common/Constants/Constants.cs

```diff
@@ -5,6 +5,13 @@ namespace UKHO.ERPFacade.Common.Constants
     [ExcludeFromCodeCoverage]
     public static class Constants
     {
+        //Event types
```

Comment:
Could we move these to two separate classes, so that we have Events.S57EventType, Transformers.S57XmlTransformer etc? 'Constants' just becomes a dumping ground for unrelated things.

Reply:
Updated as suggested.